test/controllers/corrections/VatPeriodCorrectionsListControllerSpec.scala

```diff
@@ -87,6 +88,24 @@ class VatPeriodCorrectionsListControllerSpec extends SpecBase with MockitoSugar
         Period(2022, Q1)
       )
 
+      val allPeriodsModel = Seq(
+        ListItem(
+          name = "1 July to 30 September 2021",
+          changeUrl = "/pay-vat-on-goods-sold-to-eu/northern-ireland-returns-payments" + controllers.corrections.routes.VatCorrectionsListController.onPageLoad(CheckThirdLoopMode, period, index).url,
+          removeUrl = "/pay-vat-on-goods-sold-to-eu/northern-ireland-returns-payments" + controllers.corrections.routes.RemovePeriodCorrectionController.onPageLoad(NormalMode, period, index).url
```

Comment:
we could hard code these urls to make it more readable, again something to consider but up to yous

Reply:
done